src/integrationTest/java/com/mongodb/hibernate/BasicCrudIntegrationTests.java

```diff
@@ -169,6 +170,43 @@ void testSimpleDeletion() {
         }
     }
 
+    @Nested
+    class UpdateTests {
+
+        @Test
+        void testSimpleUpdate() {
+            var statementInspector = sessionFactoryScope.getStatementInspector(SQLStatementInspector.class);
+            statementInspector.clear();
+            sessionFactoryScope.inTransaction(session -> {
+                var book = new Book();
+                book.id = 1;
+                book.title = "War and Peace";
+                book.author = "Leo Tolstoy";
+                book.publishYear = 1867;
+                session.persist(book);
+                session.flush();
+
+                book.title = "Insurrection";
+                book.publishYear = 1899;
+                session.merge(book);
+            });
+
+            assertCollectionContainsOnly(
+                    BsonDocument.parse(
+                            """
+                            {"_id": 1, "author": "Leo Tolstoy", "publishYear": 1899, "title": "Insurrection"}\
```

Comment:
```suggestion
                book.title = "Resurrection";
                book.publishYear = 1899;
                session.merge(book);
            });

            assertCollectionContainsOnly(
                    BsonDocument.parse(
                            """
                            {"_id": 1, "author": "Leo Tolstoy", "publishYear": 1899, "title": "Resurrection"}\
```

minor nit, just keeping our Tolstoy records accurate 🙂

Reply:
Good catch!!!